delegatedrouting/chunker.go

```diff
@@ -21,7 +21,6 @@ type chunker struct {
 type cidsChunk struct {
 	ContextID []byte
 	Cids      map[cid.Cid]struct{}
-	Removed   bool
```

Comment:
Remove flag is not required anymore as chunks for removed context ids simply get deleted from the datastore

Reply:
is existing data potentially parsed into this struct?

Reply:
Existing datastores will have to be removed - they got too big (that's why I've been looking into this). As the vast majority of ads are removals, they eventually start constituting >90% of the datastore. The longer index-provider runs the bigger they get. However with the current improvement datastore will grow linearly with the number of CIDs. 

Removing datastore won't harm the index.

Reply:
just making sure that we don't end up with de-serialization issues if an older data store is parsed by newer code

Reply:
Good point - will add the field back for backward compatibility